src/Packetery/Module/Checkout.php

```diff
@@ -241,7 +241,7 @@ public function __construct(
 		$this->productCategoryEntiyFactory = $productCategoryEntityFactory;
 		$this->carrierOptionsFactory       = $carrierOptionsFactory;
 		$this->latte_engine                = $latte_engine;
-		$this->options_provider            = $options_provider;
+		$this->optionsProvider             = $optionsProvider;
```

Comment:
👍  thx

latte_engine?

Reply:
Označím tedy jako resolved a budeme řešit jako refactor v samostatné větvi.